src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1402,10 +1455,7 @@ public sealed override async Task<HttpResponseMessage> SendAsync(HttpRequestMess
                         // We received the response headers but the request body hasn't yet finished.
                         // If the connection is aborted or if we get RST or GOAWAY from server, exception will be
                         // stored in stream._abortException and propagated to up to caller if possible while processing response.
-                        _ = bodyTask.ContinueWith((t, state) => {
-                                Http2Connection c = (Http2Connection)state;
-                                if (NetEventSource.IsEnabled) c.Trace($"SendRequestBody Task failed. {t.Exception}");
-                             }, this, CancellationToken.None, TaskContinuationOptions.OnlyOnFaulted, TaskScheduler.Default);
+                        _ = LogExceptionsAsync(bodyTask);
```

Comment:
Do we need to make a similar call in the catch block above for the responseHeadersTask?  Otherwise, what happens if both tasks fault: we'll propagate the exception from the request task and will end up ignoring the response task.

Reply:
done.